README.md

```diff
@@ -1,39 +1,5 @@
-# Simple Dialer
```

Comment:
why have you updated the README file?

Reply:
Sorry, as i said, it's my first pull request. I read a website that advised to do it like that. 
It seems to be a bad practice.I won't do it again...
The fix is in dialpad.kt activity.